pkg/kudoctl/cmd/install/install.go

```diff
@@ -88,37 +83,59 @@ func installFrameworks(args []string, options *Options) error {
 	}
 
 	for _, name := range args {
-		err := installFramework(name, "", *r, indexFile, kc, options)
+		err := installFramework(name, "", *r, kc, options)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-// installFramework is the umbrella for a single framework installation that gathers the business logic
-// for a cluster and returns an error in case there is a problem
-// TODO: needs testing
-func installFramework(name, previous string, repository repo.FrameworkRepository, indexFile *repo.IndexFile, kc *kudo.Client, options *Options) error {
+// getPackageCRDs tries to look for package files resolving the framework name to:
+// - a local tar.gz file
+// - a local directory
+// - a framework name in the remote repository (default)
+// in that order. Should there exist a local folder e.g. `cassandra` it will take precedence
+// over the remote repository package with the same name.
+func getPackageCRDs(name string, options *Options, repository repo.FrameworkRepository) (*repo.PackageCRDs, error) {
+	// Local files/folder have priority
+	if _, err := os.Stat(name); err == nil {
```

Comment:
I think there should be an error and fail (ReadFileSystemPackage won't parse it correctly, right?) rather than try to fall back up the chain, since the command has a side effect that would affect the state of your Kubernetes cluster.

Reply:
> Is this always relative to the root directory the command is running in?

Or absolute - the important part is that `os.Stat` can find it.
> I think there should be an error and fail (ReadFileSystemPackage won't parse it correctly, right?

exactly. Also, I wouldn't fallback - this just makes things unnecessarily complicated.

Reply:
my comment came in after this was resolved.    I would like my comment to be considered (around ignoring FileInfo)